<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupCollectionTab</class>
 <widget class="QWidget" name="GroupCollectionTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>群组采集</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="groupWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>群组列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="groupTableWidget">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="retrieveGroupsButton">
           <property name="text">
            <string>获取群组信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startMonitoringButton">
           <property name="text">
            <string>开始监听</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopMonitoringButton">
           <property name="text">
            <string>停止监听</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="userActivityWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>用户活动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="userActivityTableWidget">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>